### PR TITLE
Create Assignment-7

### PR DESCRIPTION
**Pull request by pokizadev**

### CODE REVIEW

File: assignments/assignment-7/index.html [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+        <title>Assigment-7</title>
+        <style></style>
+    </head>
+    <body>
+        <form>
+            <label for="fullNameInput">Your Full Name</label>
+            <input
+                type="text"
+                placeholder="Full Name"
+                name="name"
```
Comment (aliramazon) [inline assignments/assignment-7/index.html]: Rename name to fullName

File: assignments/assignment-7/index.html [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+            />
+
+            <label for="dateInput">DOB</label>
+            <input type="date" name="dateOfBirth" id="dateInput" />
+            <br />
+            <label for="ageInput">Your Age</label>
+            <input type="number" name="age" id="ageInput" />
+
+            <label for="male">Male</label>
+            <input type="radio" name="gender" id="male" />
```
Comment (aliramazon) [inline assignments/assignment-7/index.html]: Add value, male and female

File: assignments/assignment-7/index.html [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+
+            <label for="secretMessageInput">Your Secret Message</label>
+            <input type="hidden" name="secretMessage" id="secretMessageInput" />
+
+            <br />
+
+            <label for="colorInput">Your Preferred color</label>
+            <input type="color" name="color" id="colorInput" />
+
+            <label for="checkboxInput">Newsletter Subscription</label>
```
Comment (aliramazon) [inline assignments/assignment-7/index.html]: change id to newsletterSubscriptionCheckbox

File: assignments/assignment-7/index.html [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+            <input
+                type="checkbox"
+                name="subscriptionCheckbox"
```
Comment (aliramazon) [inline assignments/assignment-7/index.html]: name: newsletterSubscription

File: assignments/assignment-7/index.html [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+                id="checkboxInput"
+            />
+
+            <label for="languageSelect">Preferred Language</label>
+            <select name="usersLanguage" id="languageSelect">
```
Comment (aliramazon) [inline assignments/assignment-7/index.html]: rename name: preferredLanguage

File: assignments/assignment-7/index.html [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+                <option value="Russian">Russian</option>
+                <option value="Uzbek">Uzbek</option>
+                <option value="Turkish">Turkish</option>
+            </select>
+
+            <br />
+            <hr />
+
+            <label for="profilePicture">Upload Your Profile Picture</label>
+            <input type="file" name="avatar" id="profilePicture" accept="img" />
```
Comment (aliramazon) [inline assignments/assignment-7/index.html]: rename name: profilePicture

File: assignments/assignment-7/index.html [hunk 7]
```diff
@@ -0,0 +1,139 @@
...
+
+            <label for="websiteInput">Your Location</label>
```
Comment (aliramazon) [inline assignments/assignment-7/index.html]: change id to ulrInput

File: assignments/assignment-7/index.html [hunk 8]
```diff
@@ -0,0 +1,139 @@
...
+            <input type="url" name="location" id="websiteInput" />
+
+            <label for="ratingInput">Satisfaction Rating</label>
+            <input
+                type="range"
+                name="rating"
```
Comment (aliramazon) [inline assignments/assignment-7/index.html]: change name to satisfactionRating

File: assignments/assignment-7/index.html [hunk 9]
```diff
@@ -0,0 +1,139 @@
...
+            <input type="time" name="meetingTime" id="timeInput" />
+
+            <label for="weekInput">Preferred Meeting Day</label>
+            <input type="week" name="meetingDay" id="weekInput" />
+
+            <label for="monthInput">Preferred Meeting Month</label>
+            <input type="month" name="meetingMonth" id="monthInput" />
+
+            <label for="searchInput">Search for Articles</label>
+            <input type="search" name="search" id="searchInput" />
```
Comment (aliramazon) [inline assignments/assignment-7/index.html]: change name  searchedArticles

**Review by aliramazon — CHANGES_REQUESTED** [review]

**Review by aliramazon — APPROVED** [review]

**Review by aliramazon — APPROVED** [review]

**Review by durdonadev — APPROVED** [review]

**Pull request merged**